code/datums/emotes.dm

```diff
@@ -179,6 +179,15 @@
 					to_chat(user, span_notice("You cannot [key] while stunned."))
 					return FALSE
 
+		if(flags_emote & EMOTE_ARMS_CHECK)
+			///okay snapper
+			var/mob/living/carbon/snapper = user
+			var/datum/limb/left_hand = snapper.get_limb("l_hand")
+			var/datum/limb/right_hand = snapper.get_limb("r_hand")
+			if((left_hand.limb_status & LIMB_DESTROYED) && (right_hand.limb_status & LIMB_DESTROYED))
+				to_chat(user, span_notice("You cannot [key] without an arm."))
+				return FALSE
```

Comment:
Couldnt a simplemob run this code in theory
also is it possible for right or left hand to be null

Reply:
I have no clue. I just stole this code and pasted it here, really.

Reply:
If you're porting a pr you need to link it in the pr body

Reply:
The code is from another part of our codebase, i just took the noise from polaris

Reply:
![image](https://user-images.githubusercontent.com/24830358/127300626-8346a6c7-fdc5-4d96-89bd-2e4dc9547122.png)

I stole it from there, in update_icons

Reply:
limbs cant be null because theyre never removed, theyre permanent datums on mob